src/components/patterns/icons/MoonStarsIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import {SVGProps} from "react"
+
+const MoonStarsIcon = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
I'm not sure I've ever seen the pattern of: take an icon, turn it into an svg, turn the svg into a react component, then use the react component in the layout, but now that I have, I've got to say, I kind of like it.  👍